node/node.go

```diff
@@ -268,10 +268,15 @@ func (n *Node) ProcessBatch(ctx context.Context, header *core.BatchHeader, blobs
 	log.Debug("Processing batch", "num of blobs", len(blobs))
 
 	if len(blobs) == 0 {
-		return nil, errors.New("the number of blobs must be greater than zero")
+		return nil, errors.New("ProcessBatch: number of blobs must be greater than zero")
 	}
+
+	if len(rawBlobs) == 0 {
```

Comment:
This check is unneeded; if `len(blobs) != 0` and `len(rawBlobs) == 0`, then the next check will fail.